README.md

```diff
@@ -0,0 +1,198 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] This is the default test that comes with React:

### App.test.js

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/a2923fb7-6ec3-407f-86c2-c148c35c8653)

It's failing:

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/5be95592-ef44-4519-bea1-1d6f9fb705a2)


Either include some frontend tests, or kindly remove the file. 👍